### PR TITLE
Adjust sidebar positions in news, events and blog pages to align with top of content

### LINKED ISSUE #115: [3.0] Blog sidebar should be aligned with the top of the content, not the list of blog posts
**Starter 3.0.x-dev**

The sidebar should line up with the top of the content, not the list of blog posts.

We should also make the page title and breadcrumbs full width.

cc @silverstripeux

image

### PR DESCRIPTION
**Pull request by robbieaverill**
Resolves #115

### CODE REVIEW

File: templates/CWP/CWP/PageTypes/Layout/EventPage.ss [hunk 1]
```diff
@@ -1,16 +1,18 @@
 <article class="container">
     <div class="row">
-        <section class="col-lg-8<% if not $Terms && not $RelatedPages %> offset-lg-2<% end_if %>">
-            <header class="page-header">
+        <section class="col-lg-12">
+            <div class="page-header">
```
Comment (ScopeyNZ) [inline templates/CWP/CWP/PageTypes/Layout/EventPage.ss]: What was the reasoning around changing `header` back to `div`?

Comment (robbieaverill) [reply templates/CWP/CWP/PageTypes/Layout/EventPage.ss]: I think I got confused between header and head

Comment (ScopeyNZ) [reply templates/CWP/CWP/PageTypes/Layout/EventPage.ss]: I think semantically I prefer `header`. My understanding is that it's essentially just a named div.

Comment (robbieaverill) [reply templates/CWP/CWP/PageTypes/Layout/EventPage.ss]: Have updated the other templates to match

**Pull request merged**